hatchways/team-oak
# Front-end draft

Partially resolves #79 

### What this PR does (required):
- Front-end page for pet owners to review bookings, select payment method, and pay for them.

### Screenshots / Videos (front-end only):
![Screen Shot 2022-02-02 at 4 49 16 PM](https://user-images.githubusercontent.com/96796802/152244375-4af3bbc2-70d9-458c-b872-3cfd0c744938.png)
![Screen Shot 2022-02-02 at 4 51 54 PM](https://user-images.githubusercontent.com/96796802/152244391-4bb50723-6b87-41e5-ba94-ccd5724487b3.png)

### Any information needed to test this feature (required):
- navigate to `/mysitters`, view different payments in the `current`, `past-due`, and `paid` tabs.
- change payment methods (snackbar message will display a warning if you select a card that isn't your default)
- on the `paid` tab, payment area is disabled since it's no longer applicable.

### Any issues with the current functionality:
- Integration with back-end.
- Wish that paying a `current` or `past-due` invoice would move the booking into `paid`. Not implemented yet due to time constraints.
